src/main/java/com/rackspace/ceres/app/services/QueryService.java

```diff
@@ -101,11 +122,26 @@ public QueryService(ReactiveCqlTemplate cqlTemplate,
         .checkpoint();
   }
 
-  public Flux<QueryResult> queryDownsampled(String tenant, String metricName, Aggregator aggregator,
+  public Flux<QueryResult> queryDownsampled(String tenant, String metricKey, Aggregator aggregator,
       Duration granularity, Map<String, String> queryTags,
       Instant start, Instant end) {
+    return reactiveRedisTemplate.opsForSet().members(metricKey)
```

Comment:
This looks like duplicate code for queryRaw. Lets move this code to a single function that they both can call in the query service.